cloudwatch_endpoint_sg.tf

```diff
@@ -0,0 +1,98 @@
+
+  vpc_id = aws_vpc.assessment.id
+
+  tags = merge(
+    var.tags,
+    {
+      "Name" = "CloudWatch endpoints"
+    },
+  )
+}
+
+# Allow ingress via HTTPS from the Debian Desktop security group
+resource "aws_security_group_rule" "ingress_from_debiandesktop_to_cloudwatch_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.cloudwatch.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.debiandesktop.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the gophish security group
+resource "aws_security_group_rule" "ingress_from_gophish_to_cloudwatch_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.cloudwatch.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.gophish.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the guacamole security group
+resource "aws_security_group_rule" "ingress_from_guacamole_to_cloudwatch_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.cloudwatch.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.guacamole.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the kali security group
+resource "aws_security_group_rule" "ingress_from_kali_to_cloudwatch_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.cloudwatch.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.kali.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the nessus security group
+resource "aws_security_group_rule" "ingress_from_nessus_to_cloudwatch_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.cloudwatch.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.nessus.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the PenTest Portal security group
+resource "aws_security_group_rule" "ingress_from_pentestportal_to_cloudwatch_via_https" {
+  provider = aws.provisionassessment
+
+  security_group_id        = aws_security_group.cloudwatch.id
+  type                     = "ingress"
+  protocol                 = "tcp"
+  source_security_group_id = aws_security_group.pentestportal.id
+  from_port                = 443
+  to_port                  = 443
+}
+
+# Allow ingress via HTTPS from the teamserver security group
```

Comment:
> since it's the name of the security group

That was actually my justification for making many of the references to security group names lowercase, although I don't mind changing them. :laughing:

Reply:
I realized that later, but by then the capitalization train was a-rollin'!

Reply:
:steam_locomotive:
[![Motorhead](https://img.youtube.com/vi/iCMaW6a2XFg/0.jpg)](https://www.youtube.com/watch?v=iCMaW6a2XFg)